src/sentry/conf/server.py

```diff
@@ -2255,6 +2255,9 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
 # This flag activates Spotlight Sidecar in the development environment
 SENTRY_USE_SPOTLIGHT = False
 
+# This flag activates uptime checks in the developemnt environment
+SENTRY_USE_UPTIME = False
```

Comment:
We don't have one for monitors, I wonder if we should

Reply:
Yeah I'm not sure. It's kind of a pain to have to know to enable a bunch of settings just to get a feature working, but on the other hand nicer to not have to spin up a bunch of unnecessary services